src/main/java/com/nerdwin15/stash/webhook/Notifier.java

```diff
@@ -261,7 +261,7 @@ protected String getUrl(final Repository repository, final String jenkinsBase,
 
     if(strRef != null && !omitBranchName)
       url.append(String.format(BRANCH_URL_PARAMETER, urlEncode(strRef)));
-    if(!omitHashCode)
+    if(strSha1 != null && !omitHashCode)
```

Comment:
:+1: this is necessary to avoid build failures in certain Jenkins configurations. (Git plugin 2.3.5, Workflow multibranch projects)